packages/web3-eth/types/tests/eth.tests.ts

```diff
@@ -71,6 +71,24 @@ eth.subscribe('pendingTransactions');
 // $ExpectType Promise<Subscribe<Transaction>>
 eth.subscribe('pendingTransactions', (error: Error, result: Subscribe<Transaction>) => {});
 
+// $ExpectType Providers
+eth.providers;
```

Comment:
Same here with the static and non-static properties. I think you don't have to test it here too. It's enough if you test it on the AbstractWeb3Module because the Eth module and all others are extending from this class.

Reply:
I disagree this is testing the return type of the Eth classes properties! If something changes and eth inherits is removed or something we need to tests to pick it up that this would break the types hence why you would test it on this class as well. Just my view.

Reply:
>  If something changes and eth inherits is removed or something we need to tests
This will be achieved with the unit tests I wrote. They test if the module is an instance of the AbstractWeb3Module.